solr/core/src/java/org/apache/solr/update/processor/AtomicUpdateDocumentMerger.java

```diff
@@ -259,9 +264,8 @@ public boolean doInPlaceUpdateMerge(AddUpdateCommand cmd, Set<String> updatedFie
     SolrInputDocument oldDocument = RealTimeGetComponent.getInputDocument
       (cmd.getReq().getCore(), idBytes,
        null, // don't want the version to be returned
-       true, // avoid stored fields from index
        updatedFields,
-       true); // resolve the full document
+       RealTimeGetComponent.Resolution.FULL_DOC); // get the partial document for the in-place update
```

Comment:
it is not a partial document; the original comment was correct.  Now with an enum, we don't even need a comment as the enum has a self explanatory name.